release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -65,6 +65,16 @@ For more information, see xref:../installing/installing-troubleshooting.adoc#ins
 * With this update, you can export your application in the ZIP file format to another project or cluster using the *Export application* option on the *+Add* page. 
 * With this update, you can create a Kafka event sink to receive events from a particular source and send them to a Kafka topic.
 
+[id="ocp-4-12-helm-page-improvements"]
+==== Helm page improvements
+
+In {product-title} 4.12, you can use the Helm page to:
+
+* Create Helm Releases and Repositories using the *Create* button.
```

Comment:
Reading the conversation between you and Rolfe, and it seems that since it's UI elements you should bold them ,

Create **Helm Releases** and **Repositories** using the **Create** button. 

or if they're generic terms, you can simply lowercase them.

Reply:
It seems like this phrase is using the generic term. So, I'd recommend making the terms lower-case and not bold.

Reply:
Hi folks, I had not pushed the changes because I was having a little difficulty in pushing the changes and those have been resolved for now. Kindly have a look at the changes.